Gulpfile.js

```diff
@@ -680,6 +682,18 @@ gulp.step('test-functional-local-headless-firefox-run', () => {
 
 gulp.task('test-functional-local-headless-firefox', gulp.series('build', 'test-functional-local-headless-firefox-run'));
 
+gulp.step('test-functional-remote-run', () => {
+    if (QR_CODE)
+        process.env.QR_CODE = 'true';
+
+    if (BROWSER_ALIAS)
+        process.env.BROWSER_ALIAS = BROWSER_ALIAS;
+
+    return testFunctional('test/functional/fixtures', functionalTestConfig.testingEnvironmentNames.remote, functionalTestConfig.browserProviderNames.remote);
+});
+
+gulp.task('test-functional-remote', gulp.series('build', 'test-functional-remote-run'));
```

Comment:
As I understand, this task can be executed only locally. Should we reduce the test count (run only screenshot and resize tests) to reduce test execution time?

Reply:
1. The current test filter checks the browser type, not the task type. I prefer to keep this behavior. 

2. This task is not for checking "how it works in remote browsers". Tasks on BrowserStack serves this role. So I don't want to control which test will/won't run especially for this task.

3. The primary use-case for this task is connecting a browser manually to run tests in it. E.g. for debugging when you want to keep DevTools opened before starting tests to catch up a bug that occurs when a test page is loaded. Or if you want to debug your test in a VM or in a mobile device.

Reply:
Ok